src/driver/op.rs

```diff
@@ -146,7 +145,7 @@ where
                 Poll::Ready(Completion {
                     data: me.data.take().expect("unexpected operation state"),
                     result,
-                    flags,
+                    flags: flags,
```

Comment:
keep it

Reply:
We can add this back once we decide to use it. For now, let's not.

Reply:
It is just lint waring.